### ISSUE TITLE
jitpack build failed

### ISSUE THREAD

**Issue by tangxiangpi** [ChickenHook/RestrictionBypass#8]
the lastest version 2.3 is not published on jitpack:
![image](https://user-images.githubusercontent.com/18000558/207214446-3331eccb-081f-4b88-a063-34a16fdbfc76.png)
https://jitpack.io/com/github/ChickenHook/RestrictionBypass/2.3/build.log

**Comment by SarotecK** [comment]
Thanks for ur hint. Ill check it.

**Comment by Goooler** [comment]
Still broken.

**Comment by Goooler** [comment]
@SarotecK May need to publish a new version?

**Comment by SarotecK** [comment]
yes im working on it. Was busy last weeks. Thanks for ur patience.

**Comment by SarotecK** [comment]
Should be fine now. Lets wait for the next build.

**Comment by Goooler** [comment]
https://jitpack.io/com/github/ChickenHook/RestrictionBypass/2.4/build.log

**Comment by Goooler** [comment]
We'd better migrate to MavenCentral.

**Comment by SarotecK** [comment]
Yes. Ill try to use the proper way by google: https://developer.android.com/studio/publish-library. But will continue tomorrow ;) cheers.

**Comment by sfncat** [comment]
only 2.2 can use
![image](https://github.com/ChickenHook/RestrictionBypass/assets/48142516/160de555-563e-40bf-81f8-3bcad9143aa5)

**Comment by liu-wanshun** [comment]
@SarotecK please publish new version.
Now fixed.
see https://github.com/ChickenHook/RestrictionBypass/pull/16

**Comment by allentown521** [comment]
please re-publish

**Comment by sfncat** [comment]
still fail
